workflow/scripts/phenotype_process/apply_ic_field_phenotype.py

```diff
@@ -0,0 +1,16 @@
+from skimage.io import imsave
+
+from lib.shared.file_utils import read_stack
+from lib.shared.illumination_correction import apply_ic_field
+
+# load raw image data
+raw_image_data = read_stack(snakemake.input[0])
```

Comment:
Same goes for my comments in the other PR. Seems like read_stack is only used in these two instances.

Reply:
Resolved in #20